### ISSUE TITLE
A11y: Button has no accessible name

### ISSUE THREAD

**Issue by Dannyholt** [beaverbuilder/assistant#330]
According to Lighthouse the Assistant icon/button has accessible name, see below.

![](https://p74.tr4.n0.cdn.getcloudapp.com/items/YEuRRZkg/eedb6760-f9f3-4034-a3aa-3cd373c5c701.png)

**Comment by brentjett** [comment]
This is fixed with `aria-label` in the `main` branch.
